pkg/sdk/validation/manifest/json_remote_type.go

```diff
@@ -82,6 +97,55 @@ func (v *RemoteTypeValidator) checkAdditionalRefs(ctx context.Context, entity ty
 	return res, nil
 }
 
+func (v *RemoteTypeValidator) validateBackendStorageSchema(ctx context.Context, entity types.Type) (ValidationResult, error) {
+	res := ValidationResult{}
+
+	if !entity.IsExtendingHubStorage() {
+		return res, nil
+	}
+
+	opts := []public.TypeOption{
+		public.WithTypeFilter(gqlpublicapi.TypeFilter{
+			PathPattern: ptr.String(types.GenericBackendStorageSchemaTypePath),
+		}),
+		public.WithTypeLatestRevision(public.TypeRevisionSpecFields),
+	}
+
+	gotTypes, err := v.hub.ListTypes(ctx, opts...)
```

Comment:
The proper way would be to have `FindType` method on Local Hub client and use it here. Please implement such and use here, as you basically want to get a single Type based on a path. Thanks!

Reply:
Done :+1: